### PR TITLE
Replace use of undefined term authentication verifier

### LINKED ISSUE #62: Authentication verifier definition missing
https://www.w3.org/TR/2024/WD-controller-document-20240817/#authentication
The term "authentication verifier" is used without being defined.

### PR DESCRIPTION
**Pull request by selfissued**
Fixes #62

***
<a href="https://pr-preview.s3.amazonaws.com/selfissued/controller-document/pull/81.html" title="Last updated on Sep 6, 2024, 8:48 PM UTC (2580769)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/controller-document/81/eb77908...selfissued:2580769.html" title="Last updated on Sep 6, 2024, 8:48 PM UTC (2580769)">Diff</a>

### CODE REVIEW

**Review by decentralgabe — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -1386,7 +1386,7 @@ <h2>Authentication</h2>
 do with that information.
             </p>
             <p>
-This is useful to any <em>authentication verifier</em> that needs to check to
+This is useful to any party verifying the authentication that needs to check to
 see if an entity that is attempting to [=authenticate=] is, in fact,
 presenting a valid proof of authentication. When a <em>verifier</em> receives
```
Comment (TallTed) [inline index.html:1391]:
```suggestion
This is useful to any party verifying the authentication — any <em>verifier</em> —
that needs to check whether an entity that is attempting to [=authenticate=] is
presenting a valid proof of authentication. When such a <em>verifier</em> receives
```

Comment (selfissued) [reply index.html:1391]: The point of this change was to eliminate the use of the term _verifier_ with a different meaning than the definition of _verifier_ in VCDM.  See my comment on this topic at https://github.com/w3c/controller-document/issues/48#issuecomment-2321838164.

I'd be good with this change if you delete the " — any <em>verifier</em> —" part.

**Review by msporny — APPROVED** [review]

**Review by dlongley — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (TallTed) [reply index.html:1391]: Revising to eliminate *both* uses of the term `verifier` with that meaning. Also replacing `party` the introduction of which I believe the WG reconsidered some time ago.
```suggestion
This is useful to any entity verifying authentication that needs to check
whether an entity that is attempting to [=authenticate=] is presenting a valid
proof of authentication. When such an authentication verifying entity receives
```

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**